packages/web/components/table/asset-info.tsx

```diff
@@ -0,0 +1,669 @@
+    >
+      {isFavorite || isHover ? (
+        <div className="cursor-pointer">
+          <Icon
+            id="star"
+            className={classNames(
+              isHover ? "text-osmoverse-600" : "text-wosmongton-400"
+            )}
+            onClick={(event) => {
+              event.preventDefault();
+              event.stopPropagation();
+
+              if (isFavorite) onRemoveFavorite();
+              else onSetFavorite();
+            }}
+            height={24}
+            width={24}
+          />
+        </div>
+      ) : (
+        <div style={{ height: 24, width: 24 }} />
+      )}
+      <div className="flex items-center gap-4">
+        <div className="h-10 w-10">
+          {coinImageUrl && (
+            <Image alt={coinDenom} src={coinImageUrl} height={40} width={40} />
+          )}
+        </div>
+        <div className="subtitle1 flex max-w-[200px] flex-col place-content-center">
+          <div className="flex">
+            <span className="text-white-high">{coinDenom}</span>
+          </div>
+          <span className="overflow-hidden overflow-ellipsis whitespace-nowrap text-osmoverse-400">
+            {coinName}
+          </span>
+        </div>
+      </div>
+    </div>
+  );
+};
+
+const PriceCell: AssetInfoCellComponent = ({
+  row: {
+    original: { currentPrice, priceChange24h },
+  },
+}) => (
+  <div className="flex flex-col">
+    {currentPrice && (
+      <span className="subtitle1">{currentPrice?.toString()}</span>
+    )}
+    {priceChange24h && (
+      <span
+        className={classNames("caption", {
+          "text-bullish-400":
+            priceChange24h && priceChange24h.toDec().isPositive(),
+          "text-ammelia-400":
+            priceChange24h && priceChange24h.toDec().isNegative(),
+        })}
+      >
+        {priceChange24h && priceChange24h.toDec().isPositive() ? "+" : null}
+        {priceChange24h.maxDecimals(1).toString()}
+      </span>
+    )}
+  </div>
+);
+
+const SparklineChartCell: AssetInfoCellComponent<{
+  timeFrame: CommonHistoricalPriceTimeFrame;
+}> = ({
+  row: {
+    original: { coinDenom, priceChange24h },
+  },
+  timeFrame,
+}) => {
+  const { data: recentPrices } =
+    api.edge.assets.getAssetHistoricalPrice.useQuery({
+      coinDenom,
+      timeFrame,
+    });
+
+  const recentPriceCloses = useMemo(
+    () => (recentPrices ? recentPrices.map((p) => p.close) : []),
+    [recentPrices]
+  );
+
+  if (!recentPriceCloses || recentPriceCloses.length === 0)
+    return <div className="w-20" />;
+
+  const isBullish = priceChange24h && priceChange24h.toDec().isPositive();
+  const isBearish = priceChange24h && priceChange24h.toDec().isNegative();
+
+  return (
+    <Sparkline
+      width={80}
+      height={50}
+      lineWidth={2}
+      data={recentPriceCloses}
+      color={
+        isBullish
+          ? theme.colors.bullish[400]
+          : isBearish
+          ? theme.colors.ammelia[400]
+          : theme.colors.wosmongton[200]
```

Comment:
Thoughts on using an `if` statement?

```suggestion
let color: string;
if (isBullish) {
  color = theme.colors.bullish[400];
} else if (isBearish) {
  color = theme.colors.ammelia[400];
} else {
  color = theme.colors.wosmongton[200];
}
```